packages/manager/apps/web/client/app/hosting/ftp/user/add/hosting-ftp-user-add.controller.js

```diff
@@ -22,6 +22,7 @@ angular.module('App').controller(
     $onInit() {
       this.condition = this.Hosting.constructor.getPasswordConditions();
       this.model = {
+        protocol: 'both',
```

Comment:
you can use constants for protocol values to avoid scattering them (especially in the HTML file

Reply:
You're right!